scripts/create_perf_json.py

```diff
@@ -1996,6 +2088,12 @@ def main():
                     default=0,
                     dest='verbose',
                     help='Additional output when running.')
+    ap.add_argument('--csv',
+                    '-c',
+                    default=False,
+                    action="store_true",
+                    help='Only use JSON file for metric conversion is set to False.')
```

Comment:
Small change: Should help say `if set to False.`?

Broader: I interpreted the intent of this command line argument as `--json-metrics-only`. Even with `args.csv == False` I believe this script will still load CSV content from files such as `perf-uncore-events-bdx.csv`. If so, we should be clear and name the option specifically for loading metrics from JSON metric files.

Reply:
The uncore csv file is a "legacy" code path that we don't care much now. In the future, we might want to completely remove this part. 
For this script, the TMA spreadsheet is the key input that a lot of the code logics are handling it. Now, we want to update the script to use JSON input instead of this spreadsheet as default input. I will update this help message to make it more clear in sake of completeness and remove ambiguity. But for better useability, I would prefer to keep it simple. Maybe an  alternative option name would be --tma-csv. What do you think?

Reply:
Thanks Weilin. I think `--tma-csv` is clear and directly conveys that the flag is related to metrics.

Reply:
@edwarddavidbaker, the cmd opt is updated.